channels.go

```diff
@@ -0,0 +1,134 @@
+package telebot
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// ChatInfo struct is the main chat information from telegram
+type ChatInfo struct {
+	ID       int64  `json:"id"`
+	Title    string `json:"title"`
+	Username string `json:"username"`
+	Type     string `json:"type"`
```

Comment:
[chat](https://core.telegram.org/bots/api#chat) also has `first_name` and `last_name` fields which are missing here.